### PR TITLE
feat(INT-319): update deletion support to v2 token authorization

### PR DESCRIPTION
**Pull request by yashasvibajpai**
## Description of the change

> User deletion API (OpenDSR API) has been moved to support v2 api token.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Comment by Gauravudia** [comment]
@yashasvibajpai Can you add test cases

File: src/v0/destinations/af/deleteUsers.js [hunk 1]
```diff
@@ -68,7 +77,7 @@ const userDeletionHandler = async (userAttributes, config) => {
     }
     body.status_callback_urls = filteredStatusCallbackUrlsArray;
   }
-  const endpoint = `https://hq1.appsflyer.com/gdpr/opengdpr_requests?api_token=${config.apiToken}`;
+  const endpoint = `https://hq1.appsflyer.com/api/gdpr/v1/opendsr_request`;
```
Comment (mihir-4116) [inline src/v0/destinations/af/deleteUsers.js:80]: can you add endpoint doc ref here

**Review by mihir-4116 — DISMISSED** [review]

**Comment by yashasvibajpai** [comment]
> @yashasvibajpai Can you add test cases

Looks like the test suite for deletion flow is not running itself. Working on fixing it

File: src/v0/destinations/af/deleteUsers.js (hunk 1 above)
Comment (yashasvibajpai) [reply src/v0/destinations/af/deleteUsers.js:82]: Added

**Comment by sanpj2292** [comment]
The test-cases will be fixed in another PR

**Review by sanpj2292 — APPROVED** [review]

**Review by mihir-4116 — APPROVED** [review]

**Pull request merged**